recipe/meta.yaml

```diff
@@ -284,6 +284,29 @@ outputs:
         - osgeo._osr
       commands:
         - python run_test.py
+        # Check .py GDAL utilities
+        # --help not implemented as documented for gdalattachpct.py, gdal_merge.py, gdal_retile.py, gdal_sieve.py, gdalmove.py, gdal_edit.py, gdal_pansharpen.py, gdalcompare.py, ogrmerge.py?
+        # also, --version does not work with e.g. rgb2pct.py...
+        - rgb2pct.py --help
+        - pct2rgb.py --help
+        # Check for existence of gdalattachpct.py as neither --help nor --version seem to work
+        - if not exist %PREFIX%\\Library\\bin\\gdalattachpct.py exit 1  # [win]
+        - test -f $PREFIX/bin/gdalattachpct.py  # [unix]
+        - gdal_merge.py --version
+        - gdal2tiles.py --help
+        - gdal2xyz.py --help
+        - gdal_retile.py --version
```

Comment:
Oh, dear.  Seeing:
```
gdal_retile.py --version
GDAL 3.7.3, released 2023/10/30
```
but then it seems like it's exiting with an error code.

Reply:
Yep 😩

Reply:
will be addressed per https://github.com/OSGeo/gdal/pull/8676